kahuna/public/js/search/query.html

```diff
@@ -51,7 +51,7 @@
             </label>
             </li>
 
-            <li class="search__modifier-item">
+            <li ng-if="searchQuery.canUpload" id="my-uploads-checkbox" class="search__modifier-item">
```

Comment:
Do we want to hide the myUploads search for users who can't (at this moment) upload?

Reply:
Well, I believe this permission will not be something that would be toggled on and off for a user, but a more permanent permission. However, this was not an explicit requirement. I'm happy to remove this if we don't want to hide that search.

Reply:
I don’t think we have plans of introducing this permission at the Guardian. If indeed this is more likely to be permanent for a give user at the BBC, then it makes sense in my mind to also hide this checkbox (and the dropzone effect and all other upload places/opportunities). As soon as a user is granted the permission, they would all come back anyway. Only the removal would be of (small) concern, but then we are back to the most likely use case.

Any user is always able to run this search manually anyway using `+uploader:` chip. Maybe, one day, even quicker than [by typing the full email address](https://github.com/guardian/grid/issues/539#issuecomment-404420763), too 😃.

Reply:
@paperboyo actually now that you mentioned it, I wasn't aware of this dropzone effect :sweat_smile: I'll look into deactivating that as well. Do you know of any other upload features that should be blocked as well?

Reply:
> Do you know of any other upload features that should be blocked as well?

Can’t think of anything. Thanks for taking care of dropzone (on both the search page and upload page)!